data/mods/innawood/items_generic.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "iron_ore",
+    "symbol": ",",
+    "color": "brown",
+    "name": { "str": "iron ore" },
+    "category": "spare_parts",
+    "description": "A chunk of iron rich earth.  Could be refined with a bloomery.",
+    "price": 531,
+    "bashing": 7,
+    "price_postapoc": 10,
+    "material": [ "stone" ],
```

Nitpick, but maybe change it to iron?
```suggestion
    "material": [ "iron" ],
```